rules/opentitan.bzl

```diff
@@ -279,7 +280,7 @@ sign_bin = rv_rule(
         # TODO(lowRISC/opentitan:#11199): explore other options to side-step the
         # need for this transition, in order to build the ROM_EXT signer tool.
         "platform": attr.string(default = "@local_config_platform//:host"),
-        "_tool": attr.label(
```

Comment:
looks like this rename should happen other places where _tool is referenced (like on line 243)